snippets/until-___-end.sublime-snippet

```diff
@@ -1,8 +0,0 @@
-<snippet>
-    <content><![CDATA[until ${1:condition}
-	$0
-end]]></content>
-    <tabTrigger>until</tabTrigger>
```

Comment:
this is valid in crystal too

Reply:
@bew you're right. I removed these since I felt they're not worth having as snippets. The tab trigger is `until` and the expanded snippet result in `until <cursor here>`.

If we still want these around, I'll keep them. What do you think?

Reply:
I'd keep it, because it auto-adds the `end` :stuck_out_tongue:

Reply:
Agreed. I'll add this back. 🙂